src/generic_sprite.py

```diff
@@ -20,9 +20,9 @@ def __init__(
         animation = self.load_animation(images_dir, anchor)
         super().__init__(animation, x=obj.x, y=obj.y, batch=batch, group=group)
         self.scale = scale
-        self.update()
+        self.render()
 
-    def update(self):
+    def render(self):
```

Comment:
@duthades 
I thought we decided to keep the nomenclature from the parent class and use the method name ```update``` only.

Reply:
Alright!